source/iterators.tex

```diff
@@ -4043,14 +4051,32 @@
 
 \indexlibrarymember{base}{move_iterator}%
 \begin{itemdecl}
-constexpr Iterator base() const;
+constexpr Iterator base() const &;
 \end{itemdecl}
 
 \begin{itemdescr}
+\pnum
+\constraints \tcode{Iterator} satisfies
+the \oldconcept{CopyConstructible} requirements.
```

Comment:
Should be `\libconcept` not `\oldconcept`, here and in the next paragraph.

Reply:
Use \oldconcept for "meets the _Cpp17Foo_ requirements" and use \libconcept for "models `Bar`" and "satisfies `Baz`".

Reply:
More precisely, this should be `satisfies \libconcept{CopyConstructible}.` without "the" and "requirements" - we only use "the XXXX requirements" for the old named requirement sets, not with concepts.

Reply:
Yes, the oldconcept was wrong here (my bad), but thanks to both of you for explaining the difference in wording between the two - hope I'll remember it for next time!

Fixed.